docs/api/process.md

```diff
@@ -62,6 +62,14 @@ for otherwise it is `undefined`.
 When app is started by being passed as parameter to the default app, this
 property is `true` in the main process, otherwise it is `undefined`.
 
+### `process.platform`
+
+Returns the name of the platform the app is running on. `darwin` for MacOS, `linux` for Linux and `win32` for Windows.
```

Comment:
MacOS -> macOS for consistency with the rest of the docs

Sorry, I know that is very minor. Other than that, looks great!